src/sentry/api/endpoints/organization_sentry_function_details.py

```diff
@@ -0,0 +1,61 @@
+from django.http import Http404
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases import OrganizationEndpoint
+from sentry.api.endpoints.organization_sentry_function import SentryFunctionSerializer
+from sentry.api.serializers import serialize
+from sentry.models.organization import Organization
+from sentry.models.sentryfunction import SentryFunction
+from sentry.utils.cloudfunctions import delete_function, update_function
+
+# from sentry.utils.cloudfunctions import delete_function, update_function
+
+
+class OrganizationSentryFunctionDetailsEndpoint(OrganizationEndpoint):
+    private = True
+
+    def convert_args(self, request, organization_slug, function_slug, *args, **kwargs):
+        args, kwargs = super().convert_args(request, organization_slug, *args, **kwargs)
+
+        try:
+            organization = Organization.objects.get(slug=organization_slug)
```

Comment:
@vpandiarajan20 Isn't the organization already going to be an output of `super().convert_args`? We can just use that